lib/OpenQA/WebAPI/Plugin/ObsRsync/templates/ObsRsync_index.html.ep

```diff
@@ -6,11 +6,70 @@
 
 % title 'OBS synchronization folders';
 <h2><%= 'OBS synchronization folders ' %></h2>
-<ul>
-% for my $folder (@$folders) {
-    <li><%= link_to $folder => url_for('plugin_obs_rsync_folder', folder => $folder) %></li>
-% }
-</ul>
+
+<div class="row">
+    <div class="col-sm-12">
+        %= include 'layouts/info'
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>Project</th>
+                    <th>Last sync</th>
+                    <th>Synced version</th>
+                    <th>Current OBS version</th>
+                    <th>Sync status</th>
+                    <th>Job created</th>
+                    <th>Dirty status</th>
+                    <th>Last failure</th>
+                </tr>
+            </thead>
+            <tbody>
+            % for my $folder (sort keys %$folder_info_by_name) {
+                % my $folder_ident = $folder;
+                % $folder_ident =~ s/\W//g; # remove all special characters
+                % my $folder_info = $folder_info_by_name->{$folder};
+                % my $job_id = $folder_info->{run_last_job_id};
+                % my $fail_job_id = $folder_info->{fail_last_job_id};
+                <tr id="folder_<%= $folder_ident %>">
+                    <td class='project'><%= link_to $folder => url_for('plugin_obs_rsync_folder', folder => $folder)%></td>
+                    % if ($folder_info->{run_last}) {
+                    <td class='lastsynccol'>
```

Comment:
By the way, we use almost everywhere double quotes for attributes. Why break the consistency here?